tests/boards/intel_s1000_crb/src/gpio_test.c

```diff
@@ -122,13 +120,20 @@ void gpio_thread(void *dummy1, void *dummy2, void *dummy3)
 			toggle = 1;
 		}
 
-		ret = gpio_pin_write(gpio_dev, GPIO_OUT_PIN, toggle);
+		ret = gpio_pin_set(gpio_dev, GPIO_OUT_PIN, toggle);
 		if (ret) {
-			printk("Error set " GPIO_NAME "%d!\n", GPIO_OUT_PIN);
+			printk("Error setting " GPIO_NAME "%d!\n",
+			       GPIO_OUT_PIN);
 		}
 
-		gpio_pin_read(gpio_dev, GPIO_INT_PIN, &read_val);
-		printk("Reading "GPIO_NAME"%d = %d\n", GPIO_INT_PIN, read_val);
+		ret = gpio_pin_get(gpio_dev, GPIO_INT_PIN);
+		if (ret < 0) {
```

Comment:
Since this testcase is dedicated to the specific SoC and we know that `gpio_pin_get` function implemented by gpio_dw driver always returns 0 (as most of the drivers who control on chip GPIO hardware module) from my perspective it would be OK not to do error handling.